docs/medical-api/more-info/webhooks.mdx

```diff
@@ -23,7 +23,7 @@ how the Webhook flow works, see [our Webhooks guide](/home/api-info/webhooks).
 
 ### Passing Metadata
 
-You can pass metadata to the [start document query](/medical-api/api-reference/document/start-document-query) or [start consolidated data query](/medical-api/api-reference/fhir/consolidated-data-query-post),
+You can pass metadata to the [start document query](/medical-api/api-reference/document/start-document-query), [start consolidated data query](/medical-api/api-reference/fhir/consolidated-data-query-post), or [start bulk get document url](/medical-api/api-reference/document/download-url-bulk)
```

Comment:
This list will start getting pretty long once we add more routes that support webhooks.

Why don't we just say something like `You can pass metadata to endpoints that support webhooks, ...`